<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZViewClass</class>
 <widget class="QWidget" name="ZViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../plan_fakt.qrc">
    <normaloff>:/images/table.png</normaloff>:/images/table.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="6" colspan="2">
    <widget class="QTableView" name="tbl">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="cmdAdd">
     <property name="toolTip">
      <string>Добавить</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../plan_fakt.qrc">
       <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="cmdDel">
     <property name="toolTip">
      <string>Удалить</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../plan_fakt.qrc">
       <normaloff>:/images/error.png</normaloff>:/images/error.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>163</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="cmdEdit">
     <property name="toolTip">
      <string>Изменить</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../plan_fakt.qrc">
       <normaloff>:/images/modify.png</normaloff>:/images/modify.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtFilter">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="cboFilter"/>
   </item>
   <item row="5" column="2">
    <widget class="QToolButton" name="cmdPrint">
     <property name="toolTip">
      <string>Распечатать</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../plan_fakt.qrc">
       <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="cmdReload">
     <property name="toolTip">
      <string>Обновить</string>
     </property>
     <property name="statusTip">
      <string>Обновить</string>
     </property>
     <property name="whatsThis">
      <string>Обновить</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../plan_fakt.qrc">
       <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../plan_fakt.qrc"/>
 </resources>
 <connections/>
</ui>
